treeherder/config/settings.py

```diff
@@ -50,6 +50,10 @@
 # e.g. the build size tests will alert on every commit)
 PERFHERDER_REGRESSION_THRESHOLD = 2
 
+# Only generate alerts for data newer than this time in seconds in perfherder
+# default is 2 weeks
+PERFHERDER_ALERTS_MAX_AGE = 1209600
```

Comment:
When I first read this comment, I interpreted it as "only generate alerts for data newer than this specific point in time (eg a unixtimestamp)" rather than implying that it's actually a time range/delta. Also I wonder if `timedelta(weeks=2)` would be more self-documenting? 

How about something like this?
```
# Only generate alerts in perfherder for pushes newer than this age in seconds
PERFHERDER_ALERTS_MAX_AGE = timedelta(weeks=2)
```